b2b-app/imports/ui/admin/register/form/form-schemas.js

```diff
@@ -38,45 +48,77 @@ const aboutFormSchema = new SimpleSchema(
       type: String,
       max: 250,
       label: 'Reasons for volunteering',
+      uniforms: {
+        multiline: true,
+        rows: 4,
+        placeholder: [
+          'What makes you want to to volunteer at Back2Bikes?',
+          'Have you ever done any other volunteering before?',
+          'Have you worked on bikes or something similar before?',
+        ].join('\n'),
+      },
     },
   },
   { requiredByDefault: false }
 )
 
 const contactFormSchema = new SimpleSchema({
-  name: String,
-  email: SimpleSchema.RegEx.Email,
+  name: {
+    type: String,
+  },
+  mobileNumber: {
+    type: String,
+    regEx: SimpleSchema.RegEx.Phone,
+  },
+  email: {
+    type: String,
+    regEx: SimpleSchema.RegEx.Email,
+  },
+  pinNumber: {
+    type: String,
+    min: 4,
+    max: 4,
+    uniforms: {
+      'ui:field-cols': 2,
```

Comment:
Use of columns to make the UI more compact is excellent!

Reply:
Yeah, this only specifies the width of a text field in desktop. In mobile, the fields are full width. Couldn't think of a better name but it works similar to 12 column grid layouts. You can customise it in the FieldsWrap component cols prop.